<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_user</class>
 <widget class="QDialog" name="create_user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create User</string>
  </property>
  <widget class="QPushButton" name="create">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter your password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_user">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_password">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="current_password">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Create New User</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
